user-onboarding/src/components/users.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default function Users({ details }) {
+   if (!details) {
+      return (
+         <h3>Fetching Users...</h3>
+      )
+   }
+
+   const detailsArr = new Array(details)
```

Comment:
Why are you turning the object into an array of objects? 
Details is an object {}
detailsArr is an array of objects [{}]